apps/accounts/helpers.py

```diff
@@ -0,0 +1,15 @@
+from django.conf import settings
+
+from jingo import register
+
+
+@register.function
+def fxa_config():
+    return {camel_case(key): value
```

Comment:
I definitely prefer it this way but I could see that changing in the future if we started doing the 4 space version. I think we do this most places so I'll leave it for now.